checker/manual/constant-value-checker.tex

```diff
@@ -110,9 +123,7 @@ \subsection{Compile-time execution of expressions\label{staticallyexecutable-ann
 public int foo(int a, int b) {
     return a + b;
 }
-
-public void bar() {
-    int a = 5;          // a has type:  @IntVal({5}) int
+public void bar() { int a = 5;          // a has type:  @IntVal({5}) int
```

Comment:
It would be nice to maintain the empty line before `bar` and before the declaration of variable `a`.